pkgs/by-name/de/decktape/package.nix

```
@@ -0,0 +1,45 @@
+{
+  fetchFromGitHub,
+  buildNpmPackage,
+  lib,
+  chromium,
+}:
+buildNpmPackage rec {
+  name = "decktape";
+  version = "3.14.0";
+
+  src = fetchFromGitHub {
+    owner = "astefanutti";
+    repo = "decktape";
+    rev = "v${version}";
+    hash = "sha256-V7JoYtwP7iQYFi/WhFpkELs7mNKF6CqrMyjWhxLkcTA=";
+  };
+
+  npmDepsHash = "sha256-rahrIhB0GhqvzN2Vu6137Cywr19aQ70gVbNSSYzFD+s=";
+  npmPackFlags = [ "--ignore-scripts" ];
+
+  dontNpmBuild = true;
+
+  postPatch = ''
+    export PUPPETEER_SKIP_DOWNLOAD=1
+  '';
```

(Not blocking) Why not

```suggestion
  env.PUPPETEER_SKIP_DOWNLOAD = 1;
```

---

Fixed. Makes sense

---

Heads up, it looks like you forgot to push the change

---

Thanks, pushed now